pkg/upgrade/result/v1alpha1/kubernetes_test.go

```diff
@@ -52,20 +53,44 @@ func fakeGetErrClientSet() (clientset *clientset.Clientset, err error) {
 
 func fakeClientSet(k *kubeclient) {}
 
+func fakeCreateOk(cs *clientset.Clientset, upgradeResultObj *apis.UpgradeResult,
+	namespace string) (*apis.UpgradeResult, error) {
+	return &apis.UpgradeResult{}, nil
+}
+
+func fakeCreateErr(cs *clientset.Clientset, upgradeResultObj *apis.UpgradeResult,
+	namespace string) (*apis.UpgradeResult, error) {
+	return &apis.UpgradeResult{}, errors.New("some error")
+}
+
+func fakePatch(cs *clientset.Clientset, name string, pt types.PatchType, patchObj []byte,
```

Comment:
Can we rename this to `fakePatckOk`.
I missed this in my earlier comments.

Reply:
Done